src/scss/generated/_gradients.scss

```diff
@@ -0,0 +1,8 @@
+// Generated Gradients from Supernova. Do not edit manually.
+$brand-jobs-gradient-diagonal: linear-gradient(135deg, #073256 0%, #185c95 100%) !default;
+$brand-jobs-gradient-horizontal: linear-gradient(90deg, #073256 0%, #185c95 100%) !default;
```

Comment:
I think we need to discuss these names with designers. Something like `brand-diagonal` sounds better IMO. Map then can be suffixed with `-gradients` like we have it in colors.

Reply:
Refactored.